spec/money/allocation_spec.rb

```diff
@@ -1,8 +1,8 @@
 # encoding: utf-8
 
 describe Money::Allocation do
-   describe 'given number as argument' do
-     it 'raises an error when invalid argument is given' do
+  describe 'given number as argument' do
```

Comment:
This sounds more like a `context` than a `describe`, maybe we can rename it while we're at it.

```suggestion
  context 'given number as argument' do
```